sale.py

```diff
@@ -206,6 +210,14 @@ class Sale:
     "Sale"
     __name__ = 'sale.sale'
 
+    @staticmethod
+    def default_payment_authorize_on():
```

Comment:
why should gift card (just another gateway) module decide the defaults on how the payment should be captured or authorized. Please remove.